### PR TITLE
Add attachments to each message

### PR DESCRIPTION
**Pull request by ashkan18**
# Change
Add Image and PDF attachments to conversation messages.

This is a follow up on #3173 where we added conversations to admins but in there we were not showing message attachments.

# Screenshot
image

### CODE REVIEW

**Comment by zephraph** [comment]
Can you add a little more context to the PR?

File: src/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
         alignItems="center"
         justifyContent="space-between"
       >
-        <Flex width="350px">
```
Comment (ashkan18) [inline src/Apps/Conversation/Components/ConversationSnippet.tsx]: removed to use more width when showing message snippet.

File: src/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 2]
```diff
@@ -63,11 +63,17 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
               href={`/user/conversations/${conversation.internalID}`}
               underlineBehavior="hover"
             >
-              {partnerName} - {conversationText}
```
Comment (ashkan18) [inline src/Apps/Conversation/Components/ConversationSnippet.tsx]: little formatting around message snippet and item info

File: src/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 3]
```diff
@@ -63,11 +63,17 @@ const ConversationSnippet: React.FC<ConversationSnippetProps> = props => {
...
+              <Flex flexDirection={"row"}>
```
Comment (damassi) [inline src/Apps/Conversation/Components/ConversationSnippet.tsx]: Don't need `{}` for strings

**Comment by ashkan18** [comment]
thanks for reviews, going to #mergeongreen it.

**Comment by artsyit** [comment]
:rocket: PR was released in `v25.18.0` :rocket:

**Pull request merged**